website/blog/2023-04-04-presto-hudi-deep-dive.md

```diff
@@ -0,0 +1,110 @@
+---
+author: Pratyaksh Sharma
+authorURL: https://www.linkedin.com/in/pratyaksh-sharma-60593769/
+title: Presto Hudi Deep Dive
+---
+
+With the growing popularity of the lakehouse approach, it has become increasingly important for query engines to support these new formats such as Hudi. A [previous blog](https://prestodb.io/blog/2020/08/04/prestodb-and-hudi) discusses the evolution of presto-hudi integration via hive connector at a high level. With the latest community developments, a [separate presto-hudi connector](https://github.com/prestodb/presto/issues/17006) has come up but it is not at par with the hive connector in terms of security features, caching and cost-based optimization. In this blog, we dive deeper into presto-hudi integration supported via hive connector from a developer's perspective and list down the various checkpoints where the query execution moves from presto to hudi library.
```

Comment:
Can you elaborate more on "caching and cost-based optimization" of Hive connector? Maybe, ti's something we can add in Hudi connector too. We already leverage metastore caching.

Reply:
Sure, will add single liners.